ros2pkg/ros2pkg/verb/create.py

```diff
@@ -101,12 +105,24 @@ def main(self, *, args):
                 maintainer.email = maintainer.name + '@todo.todo'
 
         cpp_node_name = None
-        if args.cpp_node_name:
-            cpp_node_name = args.cpp_node_name
-            if args.cpp_node_name == args.cpp_library_name:
-                cpp_node_name = args.cpp_node_name + '_node'
-                print('[WARNING] node name can not be equal to the library name', file=sys.stderr)
-                print('[WARNING] renaming node to %s' % cpp_node_name, file=sys.stderr)
+        cpp_library_name = None
+        if args.build_type == 'ament_cmake' or args.build_type == 'cmake':
+            if args.cpp_library_name:
+                cpp_library_name = args.cpp_library_name
+            if args.cpp_node_name:
+                cpp_node_name = args.cpp_node_name
+                if args.cpp_node_name == args.cpp_library_name:
+                    cpp_node_name = args.cpp_node_name + '_node'
+                    print('[WARNING] node name can not be equal to the library name',
+                          file=sys.stderr)
+                    print('[WARNING] renaming node to %s' % cpp_node_name, file=sys.stderr)
+
+        python_node_name = None
```

Comment:
I could see that we can rename `cpp_node_name` to only `node_name` and recycle the same variable for a python and cpp project.

Reply:
This is the only recommendation I haven't resolved. I will look into it when I have a few more cycles.

@Karsten1987 Thanks for the review.

Reply:
sounds good. I'd hold this PR back until this is resolved then.
We could maybe address that in a different, follow up PR, but I rather have that all settled in one PR than introducing new CLIs and then break them again :)

Reply:
Should be handled in d4d90b.

Thanks again for the review.